src/features/event/Services/EventService.ts

```
@@ -0,0 +1,22 @@
+        event.description       = record.description ? record.description : ""
+        event.startTime         = record.start_time ? record.start_time : ""
+        event.endTime           = record.end_time ? record.end_time : ""
+        event.location          = record.location ? record.location : ""
+        event.fee               = record.fee ? record.fee : 0
+        event.userID            = record.user_id
+        event.createdAt         = record.created_at
+
+        return event
+    }
+    
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)